plugins/modules/win_scoop.ps1

```diff
@@ -0,0 +1,230 @@
+#!powershell
+
+# Copyright: (c) 2020, Jamie Magee <jamie.magee@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.ArgvParser
+#Requires -Module Ansible.ModuleUtils.CommandUtil
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+  options = @{
+    arch        = @{ type = "str"; choices = "32bit", "64bit" }
```

Comment:
It might be nice to have this as `architecture` or at least have an alias to that to match what win_chocolatey has.

Reply:
I was torn between `arch` and `architecture` here. `arch` is what scoop itself uses. From the help text:

> -a, --arch <32bit|64bit>  Use the specified architecture, if the app supports it

But I will change it if you think it's better to have a common parameter name between the different package managers.

Reply:
You can have both, `@{ type = 'str'; choices = '32bit', '64bit'; aliases = 'arch' }`. I would just recommend the full module option matches the "convention" we already have with win_chocolatey.